CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Enh #226: Add support for auto increment in primary key column. (@terabytesoftw)
 - Enh #225: Typecast refactoring (@Tigrov)
+- Bug #229: Refactoring of `Schema::normalizeDefaultValue()` method (@Tigrov)
```

Comment:
Seems, here not only refactoring, but bugfixes:

- CURRENT_TIMESTAMP for default value was not Expression()
- fix empty strings '' in default value
- fix single quote in default value
- fix DEFAULT NULL definition

?

Reply:
Yes, rigth. Thanks